TakingITGlobal/commit2act
# [TIG-770] (Rework) Styling and layout for signup container

## Ticket(s)

* [TIG-770](https://app.clickup.com/t/9009201449/TIG-770) - Original task (mostly complete)
  * [TIG-792](https://app.clickup.com/t/9009201449/TIG-792) - Focused sub-task 
* [TIG-759](https://app.clickup.com/t/9009201449/TIG-759) - Secondary task (partially complete?)

## Description

This PR extracts changes from @klancashire's [original branch](https://github.com/TakingITGlobal/commit2act/pull/116) which ran into some merge conflict troubles. Changes have mostly been ported over to this one, with the exception of the Next/Back buttons - as there was considerable Link/Button rendering work done upstream which did not survive the merge!

Please refer to the original PR for further context - minus the Next/Back buttons.

## Testing

1. Pull down and install the branch (`npm install`)
2. Run tests locally (`npm run test:basic`)
3. Run Storybook locally (`npm run storybook`)
4. View Components > LinearProgress and confirm it renders matching designs
5. View Components > Paper and confirm the gradient matches designs
6. View Views > SignupContainer and confirm the gradient background is applied
